### ISSUE TITLE
Use TChannel v2 ping support

### ISSUE THREAD

**Issue by jcorbin** [uber/ringpop#48]
Once https://github.com/uber/tchannel/issues/124 is done, ringpop should shift to using it for direct pings rather than something over call req/res.

**Comment by mranney** [comment]
Tchannel ping doesn't deliver application payloads, which we need for SWIM. It's unfortunate that both SWIM and tchannel have something called ping, but that they aren't the same thing.

**Comment by jwolski** [comment]
there seems to be an itch to conflate / merge transport-level and ring-level semantics. i think these should remain separate concerns. pings should remain separate as should membership (ring) and peer (transport) lists.

**Comment by Raynos** [comment]
:+1: for seperate concerns.

**Comment by jwolski** [comment]
closing out. no longer relevant and Tchannel v2 is released, etc, etc.
